packages/suite/src/hooks/wallet/trading/form/common/useTradingInitializer.ts

```diff
@@ -28,7 +28,7 @@ export const useTradingInitializer = ({
                 return;
             }
 
-            if (timer.timeSpend.seconds === INVITY_API_RELOAD_QUOTES_AFTER_SECONDS) {
+            if (timer.timeSpent.seconds === INVITY_API_RELOAD_QUOTES_AFTER_SECONDS) {
```

Comment:
nit: in case we change implementation of useTimer it might not measure in full seconds 
```suggestion
            if (timer.timeSpent.seconds >= INVITY_API_RELOAD_QUOTES_AFTER_SECONDS) {
```

wdyt? If you agree, please, also update tests.

Reply:
resolving after our slack discussion